hack/test-e2e-local.sh

```diff
@@ -53,7 +53,7 @@ if [[ "$1" != "operator" ]]; then
       fi
       printf "\n127.0.0.1 api.%s.external.local.gardener.cloud\n127.0.0.1 api.%s.internal.local.gardener.cloud\n" $shoot $shoot >>/etc/hosts
     done
-    printf "\n127.0.0.1 gu-local--e2e-rotate.ingress.$seed_name.seed.local.gardener.cloud\n" >>/etc/hosts
+    printf "\n127.0.0.1 gu--local--e2e-rotate.ingress.$seed_name.seed.local.gardener.cloud\n" >>/etc/hosts
```

Comment:
This (and the few related changes) confused me a bit as it seems unrelated to the actual change at hand, but only a secondary issue being fixed as part of this pull request.

Reply:
Yes, that's actually a bug that I found on the way. For historic reasons, the technical ID prefix is only `shoot-`. We only changed it to `shoot--` later on, but there might still be shoots out in the wild which don't have the `shoot--`.